docs/subnets/how-to-upgrade.md

```diff
@@ -0,0 +1,469 @@
+      --mainnet mainnet                       apply upgrade existing mainnet deployment
+      --print                                 if true, print the manual config without prompting (for public networks only)
+      --testnet testnet                       apply upgrade existing testnet deployment (alias for `fuji`)
+
+Global Flags:
+      --log-level string   log level for the application (default "ERROR")
+```
+
+Go ahead and [deploy](./create-a-local-subnet) 
+first your Subnet if that's your case.
+
+If you already had deployed the Subnet instead, you see something like this:
+
+```shell
+avalanche subnet upgrade apply testSubnet     
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? What deployment would you like to upgrade: 
+  ▸ Existing local deployment
+```
+
+Select `Existing local deployment`. This installs the upgrade file on all nodes of your local 
+network running in the background.
+
+Et voilà. This is the output shown if all went well:
+
+```shell
+✔ Existing local deployment
+.......
+Network restarted and ready to use. Upgrade bytes have been applied to running nodes at these endpoints.
+The next upgrade will go into effect 2023-03-31 09:00:00
++-------+------------+-----------------------------------------------------------------------------------+
+| NODE  |     VM     |                                        URL                                        |
++-------+------------+-----------------------------------------------------------------------------------+
+| node1 | testSubnet | http://0.0.0.0:9650/ext/bc/2YTRV2roEhgvwJz7D7vr33hUZscpaZgcYgUTjeMK9KH99NFnsH/rpc |
++-------+------------+-----------------------------------------------------------------------------------+
+| node2 | testSubnet | http://0.0.0.0:9652/ext/bc/2YTRV2roEhgvwJz7D7vr33hUZscpaZgcYgUTjeMK9KH99NFnsH/rpc |
++-------+------------+-----------------------------------------------------------------------------------+
+| node3 | testSubnet | http://0.0.0.0:9654/ext/bc/2YTRV2roEhgvwJz7D7vr33hUZscpaZgcYgUTjeMK9KH99NFnsH/rpc |
++-------+------------+-----------------------------------------------------------------------------------+
+| node4 | testSubnet | http://0.0.0.0:9656/ext/bc/2YTRV2roEhgvwJz7D7vr33hUZscpaZgcYgUTjeMK9KH99NFnsH/rpc |
++-------+------------+-----------------------------------------------------------------------------------+
+| node5 | testSubnet | http://0.0.0.0:9658/ext/bc/2YTRV2roEhgvwJz7D7vr33hUZscpaZgcYgUTjeMK9KH99NFnsH/rpc |
++-------+------------+-----------------------------------------------------------------------------------+
+```
+
+There is only so much the tool can do here for you. 
+It installed the upgrade bytes *as-is* as you configured resp. provided them to the tool.
+You should verify yourself that the upgrades were actually installed correctly, 
+for example issuing some transactions - mind the timestamp!.
+
+
+## Apply the Upgrade to a Public Node (Fuji or Mainnet)
```

Comment:
This hasn't been released yet, right? So we will need to hold this PR until we do?

Reply:
I guess it's been merged, so we put it out as soon as we do our next release

Reply:
OK so I will leave in here, just we need to wait for the merge until avalanche-cli has a new release (upcoming) @yulin-dong